pkg/ffcapi/api.go

```diff
@@ -134,6 +134,13 @@ func (eid *EventID) ProtocolID() string {
 	return fmt.Sprintf("%.12d/%.6d/%.6d", eid.BlockNumber, eid.TransactionIndex, eid.LogIndex)
 }
 
+func ProtocolIDForReceipt(receipt *TransactionReceiptResponse) string {
```

Comment:
I think this is a duplicate from https://github.com/hyperledger/firefly-transaction-manager/pull/43 - so you'll need an update once that merges.

Reply:
Yeah I used the same function from #43 knowing we would be merging around the same time. Happy to do the merge once #43 is in.